numba/tests/test_looplifting.py

```diff
@@ -576,7 +577,8 @@ def foo(X):
 
         f = jit(forceobj=True)(foo)
         f(1)
-        self.assertEqual(len(f.overloads[f.signatures[0]].lifted), 1)
+        # Post 3.12 the second loop is also being lifted
+        self.assertEqual(len(f.overloads[f.signatures[0]].lifted), 2)
```

Comment:
This change is "dead" by virtue of the skip?

Reply:
Fixed in 50ba3b38d1fa7e67cd0bb13e6489d88a691a88f0

Reply:
@esc, this should have been reverted back to `self.assertEqual(len(f.overloads[f.signatures[0]].lifted), 1)` instead of the line removed.